pkg/client/opts.go

```diff
@@ -132,6 +134,8 @@ func v2MetaHeaderFromOpts(options callOptions) *v2session.RequestMetaHeader {
 		meta.SetBearerToken(options.bearer.ToV2())
 	}
 
+	meta.SetSessionToken(options.session.ToV2())
```

Comment:
Do we need that? Right now session token already processed in `attachV2SessionToken()` function.

Reply:
Yes, `attachV2SessionToken` is called for object operations only.